src/cdk-experimental/ui-patterns/behaviors/event-manager/mouse-event-manager.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {
+  EventHandlerConfig,
+  EventHandlerOptions,
+  EventManager,
+  hasModifiers,
+  ModifierKey,
+} from './event-manager';
+
+/**
+ * The different mouse buttons that may appear on a mouse event.
+ */
+export enum MouseButton {
+  Main = 0,
+  Auxiliary = 1,
+  Secondary = 2,
+}
+
+/**
+ * A config that specifies how to handle a particular mouse event.
+ */
+export interface MouseEventHandlerConfig extends EventHandlerConfig<MouseEvent> {
+  button: number;
+  modifiers: number | number[];
+}
+
+/**
+ * An event manager that is specialized for handling mouse events. By default this manager stops
+ * propagation and prevents default on all events it handles.
+ */
+export class MouseEventManager extends EventManager<MouseEvent> {
```

Comment:
Should this be more geared towards _pointer_ events (mouse + touch) than just mouse events?